dash_bio/component_factory/_clustergram.py

```diff
@@ -45,131 +45,120 @@ def Clustergram(
     final plot.
-- standardize (string; optional): The dimension for standardizing
-    values, so that the mean is 0 and the standard deviation is 1
+- standardize (string; default 'none'): The dimension for standardizing
+    values, so that the mean is 0 and the standard deviation is 1,
     along the specified dimension: 'row', 'column', or 'none'.
-    (Default: 'none')
-- cluster (string; optional): The dimension along which the data will
-    be clustered: 'row', 'column', or 'all'; 'all' means data to be
+- cluster (string; default 'all'): The dimension along which the data will
+    be clustered: 'row', 'column', or 'all'; 'all' means data will be
     clustered along columns, then clustered along rows of
-    row-clustered data. (Default: 'all')
-- row_dist (function; optional): Function specifying the distance
-    metric for rows that will be passed to the function specified in
-    dist_fun (see scipy.spatial.distance.pdist). (Default:
-    'euclidean')
-- col_dist (function; optional): Function specifying the distance
-    metric for columns that will be passed to the function specified
-    in dist_fun (see scipy.spatial.distance.pdist). (Default:
-    'euclidean')
-- dist_fun (function; optional): Function to compute the pairwise
-    distance from the observations (see
-    scipy.spatial.distance.pdist). (Default: scs.distance.pdist)
-- link_fun (function; optional): Function to compute the linkage
-    matrix from the pairwise distances (see
-    scipy.cluster.hierarchy.linkage). (Default: sch.linkage)
-- color_threshold (dict; optional): Maximum linkage value for which
-    unique colors are assigned to clusters; 'row' for rows, and 'col'
-    for columns. (Default: {'row': 0, 'col': 0})
-- optimal_leaf_order (bool; optional): Enabling/disabling of the
-    option to determine leaf order that maximizes similarity between
-    neighboring leaves. (Default: False)
-- color_map (list; optional): The colorscale for the heatmap. Each
-    list element contains two elements; the first element refers to
-    the portion of the maximum data point under which a cell will be
-    colored, and the second element refers to the color. e.g., a
-    colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0, 'black']] mean
-    that for all cells with a value in the 50th or lower percentile of
-    the dataset, the color on the heatmap would be white; all cells
-    with a value in the 50th or higher percentile, excluding the 100th
-    percentile, would be gray; and the cell(s) in the 100th percentile
-    would be colored black. (Default: [[0.0, 'rgb(255,0,0)'], [0.5,
-    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']])
+    column-clustered data.
+- row_dist (string; default 'euclidean'): Distance metric for rows.
+    Passed as argument `metric` to the function specified in `dist_fun`.
+- col_dist (string; default 'euclidean'): Distance metric for columns.
+    Passed as argument `metric` to the function specified in `dist_fun`.
```

Comment:
There's an issue here; either `row_dist` or `col_dist` can be passed as `metric` to the function specified in `dist_fun` (not both).

Reply:
There is no issue here. The clustering can be performed on both dimensions: https://github.com/plotly/dash-bio/blob/39fec92501c83e4963874125fe4d118d2417d787/dash_bio/component_factory/_clustergram.py#L728

Reply:
I mean, the understanding is ambiguous for the end user (when reading only the docstring, as I experienced it). Now that you pointed to the relevant code block (thanks :)), I can understand clearly.

Looking at the code, we can see that a given metric may be used for the row distance (speaking elliptically):
https://github.com/plotly/dash-bio/blob/39fec92501c83e4963874125fe4d118d2417d787/dash_bio/component_factory/_clustergram.py#L734
while another metric may be used for the column distance:
https://github.com/plotly/dash-bio/blob/39fec92501c83e4963874125fe4d118d2417d787/dash_bio/component_factory/_clustergram.py#L730

But the distance function is the same, it's passed once and for all to `self._dist_fun`:
https://github.com/plotly/dash-bio/blob/39fec92501c83e4963874125fe4d118d2417d787/dash_bio/component_factory/_clustergram.py#L242

(Following the logic of a row distance vs. a column distance, we could imagine a 'row distance function' vs. a 'column distance function' -- my confusion rose from that flexibility in the distance choice but not in the distance function choice.)

I think that the following addition would clear up any confusion:
```
- row_dist (string; default 'euclidean'): Distance metric for rows.
    Passed as argument `metric` to the function specified in `dist_fun`
    when called for clustering along rows.
- col_dist (string; default 'euclidean'): Distance metric for columns.
    Passed as argument `metric` to the function specified in `dist_fun`
    when called for clustering along columns.
```

Reply:
That looks good to me! :)

Reply:
Done in 077c84a698b5a :)